src/legionpe/theta/Session.php

```diff
@@ -930,6 +930,12 @@ public function joinChannel($channel, $subLevel = self::CHANNEL_SUB_NORMAL){
 		new JoinChannelQuery($this->getMain(), $this->getUid(), $channel, $subLevel);
 		return true;
 	}
+    public function getCurrentChatState() {
+        return $this->currentChatState;
+    }
+    public function setCurrentChatState() {
```

Comment:
Do you know what a setter is ... A setter is to set a property into the parameter passed into the function...